### PR TITLE
Example: Adding codegen example from blog post

### PR DESCRIPTION
**Pull request by elitan**

### CODE REVIEW

File: examples/codegen-react-apollo/nhost/config.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+metadata_directory: metadata
+services:
+  hasura:
+    version: v2.9.0-beta.1
```
Comment (plmercereau) [inline examples/codegen-react-apollo/nhost/config.yaml:4]: we're trying to pin hasura, hasura-storage and hasura-auth versions to the ones used in our cloud. Although there's no e2e test on this example, I would use the same versions as everywhere else:
```yaml
services:
  mailhog:
    port: 8025
  hasura:
    version: v2.8.0
    environment:
      hasura_graphql_enable_remote_schema_permissions: false
  auth:
    version: 0.9.1
  storage:
    version: 0.2.2
```

**Review by plmercereau — CHANGES_REQUESTED** [review]

File: examples/codegen-react-apollo/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "@apollo/client": "^3.6.9",
+    "@nhost/react": "*",
+    "@nhost/react-apollo": "*",
+    "graphql": "^16.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "scripts": {
+    "codegen": "graphql-codegen --config graphql.config.yaml --errors-only",
+    "dev": "vite",
```
[the remark this answers is not in the document]
Comment (elitan) [reply examples/codegen-react-apollo/package.json:15]: Do you mean to remove `start`? Done.

**Review by plmercereau — APPROVED** [review]

**Comment by elitan** [comment]
Hmm, getting some errors in the tests now: https://github.com/nhost/nhost/runs/7112457367?check_suite_focus=true.

Do you know what I need to do @plmercereau ?

**Comment by plmercereau** [comment]
> Hmm, getting some errors in the tests now: https://github.com/nhost/nhost/runs/7112457367?check_suite_focus=true.
> 
> Do you know what I need to do @plmercereau ?

propably a problem with the lock file. I cleaned it and built all packages locally, it seems to have solved the problem

**Pull request merged**